src/emulator/host/src/host.cpp

```diff
@@ -112,9 +111,10 @@ bool init(HostState &state, std::uint32_t window_width, std::uint32_t border_wid
     };
 
     state.base_path = base_path.get();
-    state.pref_path = pref_path.get();
+    state.pref_path = state.base_path;
```

Comment:
Base path will be inside the application bundle on macOS. Let's keep the pref_path as-is for now, and in another commit, change pref path to current directory.

Reply:
>Base path will be inside the application bundle on macOS.
I thought you said that was ok?